andela/travel_tool_back
# #160815252 Mark single notification as read

#### What does this PR do?
Marks a single notifcation as read

#### Description of Task to be completed?
- make put request to change the value of notificationStatus to read

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Enter into the project directory - `cd travel_tool_back`
- Install dependencies and start the server - `make start`
- Make yourself a manager
- Create a new request with yourself as the manager
- Send a `PUT` request to `/api/v1/notifications/<notificationId>` (`notificationId` is the id of the notification that resulted from you creating a new travel request)
- The notification status should become `read`

#### Any background context you want to provide?
The corresponding front end PR can be found [here](https://github.com/andela/travel_tool_front/pull/167)

#### What are the relevant pivotal tracker stories?
[#160815252](https://www.pivotaltracker.com/story/show/160815252)

#### Screenshots (if appropriate)
None

#### Questions:
None
